monai/data/wsi_reader.py

```diff
@@ -358,6 +367,7 @@ def get_metadata(self, patch: np.ndarray, location: Tuple[int, int], size: Tuple
         """
         metadata: Dict = {
             "backend": "cucim",
+            "wsi_path": wsi.path,
```

Comment:
May I know why add the `wsi_path` item in the cuCIM metadata?
1. Maybe we should also add it in the `openSlide` reader?
2. Should we define the `metadata` keys in the base class to ensure different readers output the same format?

Thanks.

Reply:
I agree with both. For 1, I have already add it, and will do 2.

Reply:
The reason for adding WSI path is to know which WSI the patches is extracted from so that we can combine the patches in the related WSI.

Reply:
All done!